src/app/settings/WishListSettings.tsx

```diff
@@ -29,18 +30,26 @@ function mapStateToProps(state: RootState): StoreProps {
     numWishListRolls: state.wishLists.wishListAndInfo.wishListRolls.length,
     title: state.wishLists.wishListAndInfo.title,
     description: state.wishLists.wishListAndInfo.description,
-    wishListSource: state.settings.wishListSource
+    wishListSource: state.settings.wishListSource,
+    wishListLastUpdated: state.wishLists.lastFetched
   };
 }
 
 interface State {
   wishListSource?: string;
+  wishListLastUpdated?: Date;
 }
 
 class WishListSettings extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
-    this.state = { wishListSource: props.wishListSource };
+    this.state = {
+      wishListSource: props.wishListSource,
+      // the prop's coming over as a string instead of a date
+      wishListLastUpdated: props.wishListLastUpdated
+        ? new Date(props.wishListLastUpdated)
+        : undefined
```

Comment:
You shouldn't put this into state - you did it for the source because you wanted to have an editable working copy, but `wishListLastUpdated` is purely read from redux. Also, if you want this to be a Date, you should probably convert it in the reducer when you're storing it.